### DISCUSSION TITLE
<model-viewer> can't load glTF model hosted on google drive / dropbox

### DISCUSSION THREAD

**Discussion by eyeSphere** [google/model-viewer#2207] [Q&A]
Hello!
I'm trying to load a glTF model on a website using model-viewer.
The model is hosted on google drive and dropbox, but neither of the links load the model.
The loading bar appears and then blank.
But the model loads fine on the glTF online previewer tool.
Does anyone know what may be the issue?
Here's the example:

```
<script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.js"></script>
<script nomodule src="https://unpkg.com/@google/model-viewer/dist/model-viewer-legacy.js"></script>
<model-viewer src="https://www.dropbox.com/s/2c6vj30g9rl4olh/out.gltf"></model-viewer>
```

**Comment by elalish** [comment]
Did you check the JS console for errors and warnings? I would guess the problem is not having the right CORS headers set.

**Comment by kauhanasouza** [comment]
It`s worked? I have this issue too
